src/SingleMovie/SingleMovie.css

```diff
@@ -0,0 +1,22 @@
+.single-movie {
```

Comment:
Nice job getting a basic layout for this! We can add styling to the button later. :)